cmake/sample_defs/arch_build_custom.cmake

```diff
@@ -26,13 +26,15 @@
 # and uses the same warning options that are applied at the mission level.
 #
 add_compile_options(
-    -std=c99                # Target the C99 standard (without gcc extensions)
-    -pedantic               # Issue all the warnings demanded by strict ISO C
-    -Wall                   # Warn about most questionable operations
-    -Wstrict-prototypes     # Warn about missing prototypes
-    -Wwrite-strings         # Warn if not treating string literals as "const"
-    -Wpointer-arith         # Warn about suspicious pointer operations
-    -Wcast-align            # Warn about casts that increase alignment requirements
-    -Werror                 # Treat warnings as errors (code should be clean)
+    -std=c99                    # Target the C99 standard (without gcc extensions)
+    -pedantic                   # Issue all the warnings demanded by strict ISO C
+    -Wall                       # Warn about most questionable operations
+    -Wstrict-prototypes         # Warn about missing prototypes
+    -Wwrite-strings             # Warn if not treating string literals as "const"
+    -Wpointer-arith             # Warn about suspicious pointer operations
+    -Wcast-align                # Warn about casts that increase alignment requirements
+    -Werror                     # Treat warnings as errors (code should be clean)
```

Comment:
Moves the comments to the right 1 tab.

Reply:
Recommend avoiding tabs since they are not consistently represented in different editors.  Spaces preferred.

Reply:
Changes on all of the comments now involve spaces instead of tabs.